projects/addon-doc/components/example/example-get-tabs.pipe.ts

```diff
@@ -6,7 +6,10 @@ import {Pipe} from '@angular/core';
     name: 'tuiDocExampleGetTabs',
 })
 export class TuiDocExampleGetTabsPipe implements PipeTransform {
-    public transform(content: Record<string, string>, defaultTab: string): string[] {
+    public transform(
+        content: Record<string, never> | Record<string, string>,
```

Comment:
Why do we need to extend this type ?

I've tried to remove it – `nx build demo` still succeeds

Reply:
fixed